docs/source/ucx-debug.rst

```diff
@@ -18,7 +18,8 @@ System Configuration
     mlx5_2 port 1 ==> ib2 (Up)
     mlx5_3 port 1 ==> ib3 (Up)
 
-``ucx_info -d`` and ``ucx_info -p -u t`` are helpful commands to display what UCX understands about the underlying hardware
+``ucx_info -d`` and ``ucx_info -p -u t`` are helpful commands to display what UCX understands about the underlying hardware.
+For example, we can check
```

Comment:
Something seems to be missing here, could you check this @quasiben ?

Reply:
yup, left off my examples.  included them in the last commit